<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="This function handles multiple quality control methods for single-cell RNA-seq data."><title>Run cell-level quality control for single cell RNA-seq data. — RunCellQC • SCP</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Run cell-level quality control for single cell RNA-seq data. — RunCellQC"><meta property="og:description" content="This function handles multiple quality control methods for single-cell RNA-seq data."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">SCP</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.5.6</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/zhanghao-njmu/SCP">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Run cell-level quality control for single cell RNA-seq data.</h1>
      <small class="dont-index">Source: <a href="https://github.com/hermespara/SCP_HP/blob/HEAD/R/SCP-cellqc.R" class="external-link"><code>R/SCP-cellqc.R</code></a></small>
      <div class="d-none name"><code>RunCellQC.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>This function handles multiple quality control methods for single-cell RNA-seq data.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">RunCellQC</span><span class="op">(</span></span>
<span>  <span class="va">srt</span>,</span>
<span>  assay <span class="op">=</span> <span class="st">"RNA"</span>,</span>
<span>  split.by <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  return_filtered <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  qc_metrics <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"doublets"</span>, <span class="st">"outlier"</span>, <span class="st">"umi"</span>, <span class="st">"gene"</span>, <span class="st">"mito"</span>, <span class="st">"ribo"</span>, <span class="st">"ribo_mito_ratio"</span>,</span>
<span>    <span class="st">"species"</span><span class="op">)</span>,</span>
<span>  db_method <span class="op">=</span> <span class="st">"scDblFinder"</span>,</span>
<span>  db_rate <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  db_coefficient <span class="op">=</span> <span class="fl">0.01</span>,</span>
<span>  outlier_threshold <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"log10_nCount:lower:2.5"</span>, <span class="st">"log10_nCount:higher:5"</span>,</span>
<span>    <span class="st">"log10_nFeature:lower:2.5"</span>, <span class="st">"log10_nFeature:higher:5"</span>, <span class="st">"featurecount_dist:lower:2.5"</span><span class="op">)</span>,</span>
<span>  outlier_n <span class="op">=</span> <span class="fl">1</span>,</span>
<span>  UMI_threshold <span class="op">=</span> <span class="fl">3000</span>,</span>
<span>  gene_threshold <span class="op">=</span> <span class="fl">1000</span>,</span>
<span>  mito_threshold <span class="op">=</span> <span class="fl">20</span>,</span>
<span>  mito_pattern <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"MT-"</span>, <span class="st">"Mt-"</span>, <span class="st">"mt-"</span><span class="op">)</span>,</span>
<span>  mito_gene <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  ribo_threshold <span class="op">=</span> <span class="fl">50</span>,</span>
<span>  ribo_pattern <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"RP[SL]\\d+\\w{0,1}\\d*$"</span>, <span class="st">"Rp[sl]\\d+\\w{0,1}\\d*$"</span>,</span>
<span>    <span class="st">"rp[sl]\\d+\\w{0,1}\\d*$"</span><span class="op">)</span>,</span>
<span>  ribo_gene <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  ribo_mito_ratio_range <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">1</span>, <span class="cn">Inf</span><span class="op">)</span>,</span>
<span>  species <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  species_gene_prefix <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  species_percent <span class="op">=</span> <span class="fl">95</span>,</span>
<span>  seed <span class="op">=</span> <span class="fl">11</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>srt</dt>
<dd><p>A Seurat object.</p></dd>


<dt>assay</dt>
<dd><p>The name of the assay to be used for doublet-calling. Default is "RNA".</p></dd>


<dt>split.by</dt>
<dd><p>Name of the sample variable to split the Seurat object. Default is NULL.</p></dd>


<dt>return_filtered</dt>
<dd><p>Logical indicating whether to return a cell-filtered Seurat object. Default is FALSE.</p></dd>


<dt>qc_metrics</dt>
<dd><p>A character vector specifying the quality control metrics to be applied. Default is
`c("doublets", "outlier", "umi", "gene", "mito", "ribo", "ribo_mito_ratio", "species")`.</p></dd>


<dt>db_method</dt>
<dd><p>Doublet-calling methods used. Can be one of <code>scDblFinder</code>, <code>Scrublet</code>, <code>DoubletDetection</code>, <code>scds_cxds</code>, <code>scds_bcds</code>, <code>scds_hybrid</code></p></dd>


<dt>db_rate</dt>
<dd><p>The expected doublet rate. By default this is assumed to be 1% per thousand cells captured (so 4% among 4000 thousand cells), which is appropriate for 10x datasets.</p></dd>


<dt>db_coefficient</dt>
<dd><p>The coefficient used to calculate the doublet rate. Default is 0.01. Doublet rate is calculated as`ncol(srt) / 1000 * db_coefficient`</p></dd>


<dt>outlier_threshold</dt>
<dd><p>A character vector specifying the outlier threshold. Default is
`c("log10_nCount:lower:2.5", "log10_nCount:higher:5", "log10_nFeature:lower:2.5", "log10_nFeature:higher:5", "featurecount_dist:lower:2.5")`. See <a href="https://rdrr.io/pkg/scuttle/man/isOutlier.html" class="external-link">isOutlier</a>.</p></dd>


<dt>outlier_n</dt>
<dd><p>Minimum number of outlier metrics that meet the conditions for determining outlier cells. Default is 1.</p></dd>


<dt>UMI_threshold</dt>
<dd><p>UMI number threshold. Cells that exceed this threshold will be considered as kept. Default is 3000.</p></dd>


<dt>gene_threshold</dt>
<dd><p>Gene number threshold. Cells that exceed this threshold will be considered as kept. Default is 1000.</p></dd>


<dt>mito_threshold</dt>
<dd><p>Percentage of UMI counts of mitochondrial genes. Cells that exceed this threshold will be considered as discarded. Default is 20.</p></dd>


<dt>mito_pattern</dt>
<dd><p>Regex patterns to match the mitochondrial genes. Default is `c("MT-", "Mt-", "mt-")`.</p></dd>


<dt>mito_gene</dt>
<dd><p>A defined mitochondrial genes. If features provided, will ignore the <code>mito_pattern</code> matching. Default is <code>NULL</code>.</p></dd>


<dt>ribo_threshold</dt>
<dd><p>Percentage of UMI counts of ribosomal genes. Cells that exceed this threshold will be considered as discarded. Default is 50.</p></dd>


<dt>ribo_pattern</dt>
<dd><p>Regex patterns to match the ribosomal genes. Default is `c("RP[SL]\d+\w0,1\d*$", "Rp[sl]\d+\w0,1\d*$", "rp[sl]\d+\w0,1\d*$")`.</p></dd>


<dt>ribo_gene</dt>
<dd><p>A defined ribosomal genes. If features provided, will ignore the <code>ribo_pattern</code> matching. Default is <code>NULL</code>.</p></dd>


<dt>ribo_mito_ratio_range</dt>
<dd><p>A numeric vector specifying the range of ribosomal/mitochondrial gene expression ratios for ribo_mito_ratio outlier cells. Default is c(1, Inf).</p></dd>


<dt>species</dt>
<dd><p>Species used as the suffix of the QC metrics. The first is the species of interest. Default is <code>NULL</code>.</p></dd>


<dt>species_gene_prefix</dt>
<dd><p>Species gene prefix used to calculate QC metrics for each species. Default is <code>NULL</code>.</p></dd>


<dt>species_percent</dt>
<dd><p>Percentage of UMI counts of the first species. Cells that exceed this threshold will be considered as kept. Default is 95.</p></dd>


<dt>seed</dt>
<dd><p>Set a random seed. Default is 11.</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>Returns Seurat object with the QC results stored in the meta.data slot.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"pancreas_sub"</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">pancreas_sub</span> <span class="op">&lt;-</span> <span class="fu">RunCellQC</span><span class="op">(</span><span class="va">pancreas_sub</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Total cells: 1000 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Cells which are filtered out: 46 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 32 potential doublets </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 14 outlier cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 low-UMI cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 low-gene cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 high-mito cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 high-ribo cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 ribo_mito_ratio outlier cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 species-contaminated cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Remained cells after filtering: 954 </span>
<span class="r-in"><span><span class="fu"><a href="CellStatPlot.html">CellStatPlot</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  srt <span class="op">=</span> <span class="va">pancreas_sub</span>,</span></span>
<span class="r-in"><span>  stat.by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"db_qc"</span>, <span class="st">"outlier_qc"</span>, <span class="st">"umi_qc"</span>, <span class="st">"gene_qc"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"mito_qc"</span>, <span class="st">"ribo_qc"</span>, <span class="st">"ribo_mito_ratio_qc"</span>, <span class="st">"species_qc"</span></span></span>
<span class="r-in"><span>  <span class="op">)</span>,</span></span>
<span class="r-in"><span>  plot_type <span class="op">=</span> <span class="st">"upset"</span>, stat_level <span class="op">=</span> <span class="st">"Fail"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>stat_type is forcibly set to 'count' when plot sankey, chord, venn or upset</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> `geom_line()`: Each group consists of only one observation.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Do you need to adjust the <span style="color: #00BB00;">group</span> aesthetic?</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> `geom_line()`: Each group consists of only one observation.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Do you need to adjust the <span style="color: #00BB00;">group</span> aesthetic?</span>
<span class="r-plt img"><img src="RunCellQC-1.png" alt="" width="700" height="433"></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">pancreas_sub</span><span class="op">$</span><span class="va">CellQC</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Pass Fail </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  954   46 </span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"ifnb_sub"</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">ifnb_sub</span> <span class="op">&lt;-</span> <span class="fu">RunCellQC</span><span class="op">(</span><span class="va">ifnb_sub</span>, split.by <span class="op">=</span> <span class="st">"stim"</span>, UMI_threshold <span class="op">=</span> <span class="fl">1000</span>, gene_threshold <span class="op">=</span> <span class="fl">550</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> === CTRL ===</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Total cells: 1000 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Cells which are filtered out: 308 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 47 potential doublets </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 8 outlier cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 28 low-UMI cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 250 low-gene cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 high-mito cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 high-ribo cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 ribo_mito_ratio outlier cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 species-contaminated cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Remained cells after filtering: 692 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> === STIM ===</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Total cells: 1000 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Cells which are filtered out: 302 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 41 potential doublets </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 12 outlier cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 25 low-UMI cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 251 low-gene cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 high-mito cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 high-ribo cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 ribo_mito_ratio outlier cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> ... 0 species-contaminated cells </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> &gt;&gt;&gt; Remained cells after filtering: 698 </span>
<span class="r-in"><span><span class="fu"><a href="CellStatPlot.html">CellStatPlot</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  srt <span class="op">=</span> <span class="va">ifnb_sub</span>,</span></span>
<span class="r-in"><span>  stat.by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"db_qc"</span>, <span class="st">"outlier_qc"</span>, <span class="st">"umi_qc"</span>, <span class="st">"gene_qc"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"mito_qc"</span>, <span class="st">"ribo_qc"</span>, <span class="st">"ribo_mito_ratio_qc"</span>, <span class="st">"species_qc"</span></span></span>
<span class="r-in"><span>  <span class="op">)</span>,</span></span>
<span class="r-in"><span>  plot_type <span class="op">=</span> <span class="st">"upset"</span>, stat_level <span class="op">=</span> <span class="st">"Fail"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>stat_type is forcibly set to 'count' when plot sankey, chord, venn or upset</span>
<span class="r-plt img"><img src="RunCellQC-2.png" alt="" width="700" height="433"></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">ifnb_sub</span><span class="op">$</span><span class="va">CellQC</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Pass Fail </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1390  610 </span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Hao Zhang.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer></div>

  

  

  </body></html>

